"Development" page complete

anasxyz opened the pull request:
I created a navigation bar and put some of the wikipedia text in, I know there's no background or images or anything yet and it probably doesn't look like much right now but that's because I put more time into the navigation bar than other stuff. let me know if it's good or not, I'm also planning to add sliding images maybe, idk yet. I'll continue working on it this week.

Screenshot:
![screenshot](https://user-images.githubusercontent.com/114344706/197911592-12220976-5c8f-42e8-8b78-1e2ff8a779b3.PNG)

arvin-shafiei commented on development_page.html:
```diff
@@ -0,0 +1,148 @@
+    <header class="sticky-top">
+        <!-- Nav bar -->
+        <nav class="navbar navbar-expand-sm navbar-dark p-3">
+            <div class="collapse navbar-collapse pb-2" id="navbarNav">
```
If you put the collapse class, it makes it so if the width of the display goes under 576px the navbar disappears.

```suggestion
            <div class="navbar-collapse pb-2" id="navbarNav">
```

This will stop it from disappearing and make the navbar responsive.

anasxyz replied:
Yes i didn't notice that, i kept wondering why it was disappearing everytime i made the window small.. thank you.

arvin-shafiei commented on development_page.html:
```diff
@@ -0,0 +1,148 @@
+                        <a class="nav-link" href="#">GAMEPLAY</a>
+                    </li>
+                    <li class="nav-item px-3 active pdfont">
+                        <a class="nav-link" href="#">DEVELOPMENT</a>
```
```suggestion
                        <a class="nav-link" href="development_page.html">DEVELOPMENT</a>
```
Maybe also make the href direct to the development page on the navbar since the HTML has been made for it.

anasxyz replied:
Yes ur right

anasxyz commented:
This is how it looks so far. I've added sliding images, changed some of the text around, and put background images. There's also still the "Production" area at the bottom and other small details that I need to touch up on and finish, which will hopefully be done within a day or two. Let me know if I should change something or if something doesn't look good.

![image](https://user-images.githubusercontent.com/114344706/198906463-1bae74e6-bb7f-46d4-90a3-72b13aaeff43.png)

ke444a commented:
I think that the page is really well-done. What do you think about squeezing text a little bit by putting it into .container instead of .container fluid, or adding some additional padding inside your box with text? I am not sure but it may look nicer if the text is less spread across web page.

anasxyz commented:
> I think that the page is really well-done. What do you think about squeezing text a little bit by putting it into .container instead of .container fluid, or adding some additional padding inside your box with text? I am not sure but it may look nicer if the text is less spread across web page.

I had it in .container only at first and I was trying to see if I looked better with .container-fluid. I will revert it back to .container to see how it looks.

ke444a reviewed (approved):
I think that page looks really nice, along with a well-structured and correct HTML & CSS file. It is ready to be merged.
